check_process

```diff
@@ -16,17 +16,17 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		backup_restore=0
-		multi_instance=0
+		backup_restore=1
+		multi_instance=1
 		incorrect_path=1
 		port_already_use=0
 		change_url=0
 ;;; Levels
 	Level 1=auto
 	Level 2=auto
 	Level 3=auto
-# Level 4:
-	Level 4=0
+# Level 4: the tool can't talk with LDAP
+	Level 4=1
```

Comment:
Imho it shouldn't be 1 here ... It should either be 0 or n/a ... This app has user accounts, and those accounts could be linked to LDAP if an LDAP interface was implemented. It's not there yet, but putting 1 here, imho, means "this app is correctly interfaced with LDAP". We could put n/a, but not me that would mean "The interface with LDAP is irrelevant because this app doesn't have user accounts". So to me '0' is the good value to have...

Reply:
ok, i'll try with n/a if it doesn't downgrade the level of this app
There isn't much hope to have a LDAP support soon: https://bugs.galette.eu/issues/239